packages/adapters-library/src/scripts/rpcInterceptor.ts

```diff
@@ -0,0 +1,119 @@
+import { createHash } from 'node:crypto'
+import { http, bypass } from 'msw'
+import { setupServer } from 'msw/node'
+import { Json } from '../types/json'
+
+type RpcRequest = {
+  method: string
+  params: Json[]
+  id: number
+}
+
+function createKey(url: string, requestBody: Json) {
+  return createHash('md5')
+    .update(url + JSON.stringify(requestBody))
+    .digest('hex')
+}
+
+function slimRequest({ method, params }: RpcRequest) {
+  return {
+    method,
+    params,
+  }
+}
```

Comment:
Needs to strip `id` as it's an arbitrary number on every execution, so it cannot be part of the hash.